src/pad/src/pad.tcl

```diff
@@ -145,7 +145,7 @@ sta::define_cmd_args "define_pad_cell" {[-name name] \
 
 proc define_pad_cell {args} {
   sta::parse_key_args "define_pad_cell" args \
-    keys {-name -type -cell_name -orient -pad_pin_name -break_signals} \
+    keys {-name -type -cell_name -orient -pad_pin_name -break_signals -offset} \
```

Comment:
Where is this new arg used?

Reply:
its passed to the `ICeWall::add_libcell` function on line 173, that function parses all the arguments again, so on line 5859 is were it sets the offset parameter in the cell definition.